### PR TITLE
Bump version to 1.6.0

### PR DESCRIPTION
**Pull request by hartwork**
With the slight differences in behavior and the number of improvements maybe "2.0.0" will be a better fit that "1.5.3", so I went for that. Just an idea, and also demos a pull request where UI tests are affected.

### CODE REVIEW

**Review by edgar-bonet — COMMENTED** [review]
It makes sense to bump to at least 1.6. Maybe to 2.0 if introducing a behavior incompatible with previous versions, e.g. more tolerant parsing. In the end, I think the decision belongs to @tenox7.

**Comment by tenox7** [comment]
I would prefer 1.6 rather than 2.0. I think I want to reserve 2.0 if we completely rewrite it in Go or Rust or something like that.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: ttyplot
-version: 1.5.2
+version: 2.0.0
```
Comment (tenox7) [inline snap/snapcraft.yaml:2]: I would prefer 1.6.x, thanks

**Comment by hartwork** [comment]
> I would prefer 1.6 rather than 2.0. I think I want to reserve 2.0 if we completely rewrite it in Go or Rust or something like that.

@tenox7 alright, re-targeted to 1.6.0 now.

**Review by edgar-bonet — APPROVED** [review]

**Pull request merged**